### PR TITLE
fix: #3149 - Msal Redirect Component was not part of the Module

### PR DESCRIPTION
**Pull request by roopeshreddy**
This PR fix the dangling component added to the NgModule. As per the Angular docs, any dangling component which is not part of the module or referenced in any other component will be removed during the tree shaking.

### CODE REVIEW

File: lib/msal-angular/src/msal.module.ts [hunk 1]
```diff
@@ -12,9 +12,10 @@ import { MsalGuard } from "./msal.guard";
 import { MsalBroadcastService } from "./msal.broadcast.service";
 import { MsalService } from "./msal.service";
 import { MSAL_INSTANCE , MSAL_GUARD_CONFIG, MSAL_INTERCEPTOR_CONFIG } from "./constants";
+import { MsalRedirectComponent } from './msal.redirect.component';
```
Comment (pkanher617) [inline lib/msal-angular/src/msal.module.ts:15]: Change the `'` to `"` so the linter isn't erroring out.

**Comment by jasonnutter** [comment]
@roopeshreddy Thanks for the PR! Can you please address the lint errors, and also run `npm run beachball:change` from the root of the repo, and complete the browsers and push the generated change files?

@jo-arroyo Can you please test this PR locally?

File: lib/msal-angular/src/msal.module.ts (hunk 1 above)
Comment (roopeshreddy) [reply lib/msal-angular/src/msal.module.ts]: This is taken care.

**Comment by jasonnutter** [comment]
@roopeshreddy Can you discard/undo the changes to the package-lock files?

**Comment by roopeshreddy** [comment]
> @roopeshreddy Can you discard/undo the changes to the package-lock files?

Done. I've reverted the package-local.json files

**Review by tnorling — APPROVED** [review]

**Pull request merged**